### ISSUE TITLE
UrsulaConfiguration per se reads from raw.githubusercontent.com.

### ISSUE THREAD

**Issue by jMyles** [nucypher/nucypher#1230]
I only noticed this when I tried to run the test suite while offline.

```
ERROR
test setup failed
self = <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3f69aedd68>

    def _new_conn(self):
        """ Establish a socket connection and set nodelay settings on it.
    ERROR
test setup failed
```
...
```
>           raise MaxRetryError(_pool, url, error or ResponseError(cause))
E           urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /nucypher/ethereum-contract-registry/goerli/contract_registry.json (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3f69aedd68>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))

../../../../.virtualenvs/nucypher-EzeC4Bqx/lib/python3.7/site-packages/urllib3/util/retry.py:399: MaxRetryError

During handling of the above exception, another exception occurred:

    @pytest.fixture(scope="module")
    def ursula_federated_test_config():
        ursula_config = UrsulaConfiguration(dev_mode=True,
                                            rest_port=MOCK_URSULA_STARTING_PORT,
                                            start_learning_now=False,
                                            abort_on_learning_error=True,
                                            federated_only=True,
                                            network_middleware=MockRestMiddleware(),
                                            save_metadata=False,
>                                           reload_metadata=False)

../fixtures.py:132: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../nucypher/config/characters.py:69: in __init__
    super().__init__(dev_mode=dev_mode, *args, **kwargs)
../../nucypher/config/node.py:154: in __init__
    self.initialize(password=DEVELOPMENT_CONFIGURATION)
../../nucypher/config/node.py:394: in initialize
    self.registry_filepath = EthereumContractRegistry.download_latest_publication()
../../nucypher/blockchain/eth/registry.py:94: in download_latest_publication
    response = requests.get(github_endpoint)
../../../../.virtualenvs/nucypher-EzeC4Bqx/lib/python3.7/site-packages/requests/api.py:75: in get
    return request('get', url, params=params, **kwargs)
../../../../.virtualenvs/nucypher-EzeC4Bqx/lib/python3.7/site-packages/requests/api.py:60: in request
    return session.request(method=method, url=url, **kwargs)
../../../../.virtualenvs/nucypher-EzeC4Bqx/lib/python3.7/site-packages/requests/sessions.py:533: in request
    resp = self.send(prep, **send_kwargs)
../../../../.virtualenvs/nucypher-EzeC4Bqx/lib/python3.7/site-packages/requests/sessions.py:646: in send
    r = adapter.send(request, **kwargs)
```
...
```
>           raise ConnectionError(e, request=request)
E           requests.exceptions.ConnectionError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /nucypher/ethereum-contract-registry/goerli/contract_registry.json (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3f69aedd68>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))

../../../../.virtualenvs/nucypher-EzeC4Bqx/lib/python3.7/site-packages/requests/adapters.py:516: ConnectionError
```

**Comment by szotov** [comment]
This should work properly in #1225

**Comment by vzotova** [comment]
Closed in #1225
